src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -3006,6 +3008,17 @@ private function registerHtmlSanitizerConfiguration(array $config, ContainerBuil
             $def = $container->register($configId, HtmlSanitizerConfig::class);
 
             // Base
+            if ($sanitizerConfig['default_action'] ?? false) {
+                if (InstalledVersions::satisfies(new VersionParser(), 'symfony/html-sanitizer', '>=7.2') === false) {
```

Comment:
We can check for the existence of the `HtmlSanitizerAction` class instead (the same can be done in the test).

Reply:
smart. done.